source/includes/_v2_endpoints.md

```diff
@@ -1441,3 +1441,79 @@ the query.
+
+```
+
+```shell
+# Post a link
+curl -X POST  https://api.backstit.ch/v2/organizations/9211967035420133bff950e140978a72/post?type=link&url=https%3A%2F%2Fmedium.com%2Fbackstitch-inc%2F100-industries-over-100-days-7521e75027ff&email=api_example%40backstit.ch&feed_name=Test+Source
+
+# Post an image
+curl -X POST  https://api.backstit.ch/v2/organizations/9211967035420133bff950e140978a72/post?type=image&image=https%3A%2F%2Fimages-backstitch.s3.amazonaws.com%2Fnext%2Flogos%2Fbackstitch_white_small.png&description=The+backstitch+logo+in+white&email=api_example%40backstit.ch&feed_name=Test+Source
+
+# Post a text post
+curl -X POST  https://api.backstit.ch/v2/organizations/9211967035420133bff950e140978a72/post?type=text&title=Test%20title&text=Lorem+ipsum+dolor+sit+amet%2C+consectetur+adipiscing+elit%2C+sed+do+eiusmod+tempor+incididunt+ut+labore+et+dolore+magna+aliqua.+Ut+enim+ad+minim+veniam%2C+quis+nostrud+exercitation+ullamco+laboris+nisi+ut+aliquip+ex+ea+commodo+consequat.+Duis+aute+irure+dolor+in+reprehenderit+in+voluptate+velit+esse+cillum+dolore+eu+fugiat+nulla+pariatur.+Excepteur+sint+occaecat+cupidatat+non+proident%2C+sunt+in+culpa+qui+officia+deserunt+mollit+anim+id+est+laborum&email=api_example%40backstit.ch&feed_name=Test+Source
+
+```
+
+This endpoint allows you to post to custom sources. These posts will appear in all topics that the custom source has been added to.
+
+### HTTP Request
+
+`POST https://api.backstit.ch/v2/organizations/{ORGANIZATION_KEY}/post`
+
+### URL Parameters
+
+| Parameter | Required | Description |
+|---------|:-------:|:-----------|
+| ORGANIZATION_KEY | yes | Your organization’s api key is obtained from the organization dashboard under settings. |
+
+### Query Parameters
+
+These parameters are required regardless of the post type.
+
+| Parameter | Required | Description |
+|---------|:-------:|:-------:|:-----------|
+| feed_name | yes | The name of the custom feed that will be posted to. |
+| type | yes | The type of post that is being posted to the feed (text, image, or link). Note that different parameters are required for each type of post (see below). |
+| email | yes | The email address of the user that will be credited for the post. |
+
+### Link Parameters
+
+| Parameter | Required | Description |
+|---------|:-------:|:-------:|:-----------|
+| url | yes | The URL of the article that you would like to be scraped for the post. |
+| title | no | The title of the post (if none is provided, backstitch will attempt to scrape the article's title from the source). |
```

Comment:
The title of the post.  Note: if none is provided, backstitch will attempt to scrape the article's title from the source.